### ISSUE TITLE
about AMsoftmax

### ISSUE THREAD

**Issue by DYJ1111** [clovaai/voxceleb_trainer#21]
Hi, I use same coda, just different loss function (softmax vs AMsoftmax). But the loss is reducing slowly and acc is increase slowly when i use amsoftmax .  Have you ever meet this problem?

**Comment by DYJ1111** [comment]
I use ResnetSe34 as the model

**Comment by joonson** [comment]
Yes, we have observed that it can take longer to converge and gets lower accuracy, but the verification EER is better.

**Comment by DYJ1111** [comment]
> Yes, we have observed that it can take longer to converge and gets lower accuracy, but the verification EER is better.

So could you please tell me the number of epoches? i set epoch is 40, but the loss is too high (7.x) and acc is too low(3.x%)

**Comment by 009deep** [comment]
@DYJ1111  Check out [issue10](https://github.com/clovaai/voxceleb_trainer/issues/10#issuecomment-619394722)

@joonson  one question I have for those logs is model mentioned there is 

> model ResNetSEFB34X

I am guessing it refers ResNetSE34L in this repo. If not please correct me what iis difference in that  model.

**Comment by DYJ1111** [comment]
> @DYJ1111 Check out [issue10](https://github.com/clovaai/voxceleb_trainer/issues/10#issuecomment-619394722)
> 
> @joonson one question I have for those logs is model mentioned there is
> 
> > model ResNetSEFB34X
> 
> I am guessing it refers ResNetSE34L in this repo. If not please correct me what iis difference in that model.

Thanks, I use ResNetSE34L and train 100 epoches, the TEER is 50% and veer is 4.8

**Comment by 009deep** [comment]
@DYJ1111  Results Joon Soon has presented is at 500 epochs for different hyper parameters.

**Comment by joonson** [comment]
@009deep ResNetSEFB34X is the same as ResNetSE34L in the released version
